tests/Unit/Elliptic/DiscontinuousGalerkin/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ add_test_library(
   ${LIBRARY}
   "Elliptic/DiscontinuousGalerkin/"
   "${LIBRARY_SOURCES}"
-  "DataStructures;Domain;ErrorHandling"
+  "Boost::boost;DataStructures;Domain;ErrorHandling"
```

Comment:
AFAICT these tests don't use boost, so it shouldn't be linked here. A bunch of other libraries are missing (my fault), so perhaps adding them fixes the issue you had:

```
Elliptic;Utilities;Parallel;DomainStructure;CoordinateMaps;DomainCreators;DiscontinuousGalerkin;ParallelDg;Initialization;AnalyticSolutions;Spectral;LinearOperators
```

Even if this fixes the issue I don't understand why cmake would complain about boost here, because DataStructures actually links it in publicly... We should really convert our test libs to the new cmake structure 😅